lib/rules/aria-allowed-role.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "aria-allowed-role",
+  "selector": "[role]",
+  "tags": [
```

Comment:
This should only contain `cat.aria` and `best-practice`. Rules are either a best practice or a WCAG violation. It's never both.

Reply:
Done.